R/brm.R

```diff
@@ -231,6 +231,11 @@
 #'   (e.g., initial values, number of iterations, control arguments, ...). A
 #'   known limitation is that a refit will be triggered if within-chain
 #'   parallelization is switched on/off.
+#' @param file_auto Logical If \code{TRUE} file parameter will be calculated
```

Comment:
I would call it "argument" not "parameter". Also put `file` and other arguments mentioned here in \code{}